README.md

```diff
@@ -5,6 +5,12 @@ Android app for the 4th Android Developer conference- droidcon to be held in Nai
 This project is the Android app for the conference. The app supports devices running Android 5.0+,
 and is optimized for phones and tablets of all shapes and sizes.
 
+## Running the Project
+
+To ensure the project runs on your local environment ensure to you have Java 11 on your pc or if you don't have you can install it from [here](https://www.oracle.com/java/technologies/javase/jdk11-archive-downloads.html). 
+
+If you have multiple installations of Java make sure to set Java 11 as your preferred version to run the project.
```

Comment:
Can we add steps to check if you've selected the right java version?
https://developer.android.com/studio/intro/studio-config#jdk

Reply:
Ok let me add it too